CovidTracking/CovidTracking/Extension/ViewController+rx.swift

```diff
@@ -0,0 +1,20 @@
+//
+//  ViewController+rx.swift
+//  CovidTracking
+//
+//  Created by Quang Nguyen on 5/24/21.
+//  Copyright © 2021 Tiến. All rights reserved.
+//
+
+import Foundation
+import RxSwift
+import RxCocoa
+import UIKit
+
+extension RxSwift.Reactive where Base: UIViewController {
```

Comment:
```suggestion
extension Reactive where Base: UIViewController {
```
đặt lại tên file đi em